plugins/module_utils/foreman_helper.py

```diff
@@ -87,6 +87,32 @@ def ping(self):
 
 class ForemanApypieAnsibleModule(ForemanBaseAnsibleModule):
 
+    def _patch_location_api(self):
+        """This is a workaround for the broken taxonomies apidoc in foreman."""
```

Comment:
Nitpick, can you add the link to the foreman issue number? That way we are more likely to remove this in future.

Reply:
@sean797 this should be it